### ISSUE TITLE
Add reviewers attribute to `MergeRequest`

### ISSUE THREAD

**Issue by zemzale** [xanzy/go-gitlab#1030]
Gitlab 13.5 intreduced reviewers and they have been enabled by default since 13.7 https://docs.gitlab.com/ee/user/project/merge_requests/getting_started#reviewer

There is a problem that they are not documented properly in https://docs.gitlab.com/ce/api/merge_requests.html, but it's possible to see them in Gitlab.com API. Here is an example :
```json
{
   ...
    "reviewers": [
        {
            "id": 2182404,
            "name": "Alberts Zemzale",
            "username": "zemzale",
            "state": "active",
            "avatar_url": "https://assets.gitlab-static.net/uploads/-/system/user/avatar/2182404/avatar.png",
            "web_url": "https://gitlab.com/zemzale"
        }
    ],
    ...
}
```

I would gladly add this if there are nothing blocking this

**Comment by svanharmelen** [comment]
By all means feel free to add a PR fir this 👍🏻

**Comment by zemzale** [comment]
Closed by #1032
